.github/workflows/main.yml

```diff
@@ -4,38 +4,30 @@ on:
   push:
   pull_request:
   workflow_dispatch:
+  release:
+    types: ['published', 'released']
```

Comment:
Those triggers are not needed with pyuploadtool.

Reply:
see above